app/models/patient.rb

```diff
@@ -78,11 +73,15 @@ class Patient < ApplicationRecord
             presence: { message: "is required when 'Isolation' is 'false'" },
             if: -> { !isolation }
 
-  validates :email, on: :api, email: true
+  validates :email, email: true
 
-  validates :assigned_user, numericality: { only_integer: true, allow_nil: true, greater_than: 0, less_than_or_equal_to: 999_999 }
+  validates :assigned_user, numericality: { only_integer: true,
+                                            allow_nil: true,
+                                            greater_than: 0,
+                                            less_than_or_equal_to: 999_999,
+                                            message: 'is not valid, acceptable values are numbers between 1-9999' }
```

Comment:
Message should say `1-999999`

Reply:
Addressed in https://github.com/SaraAlert/SaraAlert/pull/582/commits/9106e97af2c69a78aee72135c1c39199e6823f83.